pyproject.toml

```diff
@@ -67,6 +68,8 @@ viz = [
     "param>=2.0.0",
 ]
 
+skopt = ["scikit-optimize>=0.10.0,<0.11"]
+
```

Comment:
Is it worth to have something like `all` here which downloads all optional dependency sets?

Reply:
Done